src/features/events/components/EventForm/context.tsx

```
@@ -10,29 +11,17 @@
-
-const timezones = listTimeZones().map((timeZone) => ({
-  value: timeZone,
-  label: `${timeZone} (${getTimezoneOffsetLabel(timeZone)})`,
-}))
-
 export const getFormatedScope = (scope: RestUserScopesResponse[number]): SelectOption<string> => {
+  const { name, description } = getFormatedScopeData(scope)
   return {
     value: scope.code,
-    label: [<SF.Text semibold>{scope.name}</SF.Text>, ' ', <SF.Text>{scope.zones.map(({ name, code }) => `${name} (${code})`).join(', ')}</SF.Text>],
+    label: [<SF.Text semibold>{name}</SF.Text>, ' ', <SF.Text>{description}</SF.Text>],
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)